capplets/common/activate-settings-daemon.c

```diff
@@ -26,35 +25,41 @@ static void popup_error_message (void)
 gboolean
 activate_settings_daemon (void)
 {
-  DBusGConnection *connection = NULL;
-  DBusGProxy *proxy = NULL;
-  GError *error = NULL;
-
-  connection = dbus_g_bus_get (DBUS_BUS_SESSION, &error);
-  if (connection == NULL)
-    {
-      popup_error_message ();
-      g_error_free (error);
-      return FALSE;
-    }
-
-  proxy = dbus_g_proxy_new_for_name (connection,
-                                     "org.mate.SettingsDaemon",
-                                     "/org/mate/SettingsDaemon",
-                                     "org.mate.SettingsDaemon");
+    GError     *error = NULL;
+    GDBusProxy *proxy = NULL;
+    GVariant *ret;
 
-  if (proxy == NULL)
-    {
-      popup_error_message ();
-      return FALSE;
+    if (proxy == NULL) {
```

Comment:
This check is redundant and can be removed, since it is just initialized and the variable is not global.